x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -20,17 +20,17 @@ public class BlobCacheMetrics {
     public BlobCacheMetrics(MeterRegistry meterRegistry) {
         this(
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.miss_that_triggered_read",
+                "es.blob_cache.miss_that_triggered_read.count",
```

Comment:
Is this `total` instead?

Reply:
count and total feel very similar to me. I would use `total` for async counter or gauge (reporting a cumulative value)
and count for `deltas` like here.

Reply:
Count should for the current number of a fluctuating quantity, e.g. the number of tabs _currently_ opened in your browser is `browser.window.open.count`.
Total should be for an always increasing quantity, e.g. the number of times you have been oncall `oncall.total`

This looks like a total to me (we look at all the cache misses and we keep increasing the counter).

Reply:
as we discussed offline.
I will change `count` to be `current`. The intention is to use this for values of current snapshot/sample. That is for non-monotonic metrics (number of tabs can go up and down) Does not matter if it is additive or not. 
`total` will be always monotonic metrics (a counter). I will remove the word of `sum` as it might be confusing if the value reported should cumulative or delta.